src/store/actions.ts

```diff
@@ -50,25 +52,85 @@ export type InitParams = {
 	lexicalCategory?: SearchedItemOption;
 };
 
+interface ValidLexemeData {
+	validLemma: string;
+	validLanguageId: string;
+	validLexicalCategoryId: string;
+	validSpellingVariant: string;
+}
+
 export default function createActions(
 	lexemeCreator: LexemeCreator,
 	langCodeRetriever: LangCodeRetriever,
 	languageCodesProvider: LanguageCodesProvider,
 	tracker: Tracker,
 ): RootActions {
 	return {
-		async [ CREATE_LEXEME ]( { state, commit }: RootContext ): Promise<string> {
-			if ( !state.language || !state.lexicalCategory ) {
-				throw new Error( 'No language or lexical category!' ); // TODO
+		[ GET_VALID_INPUTS ]( { state, commit }: RootContext ): ValidLexemeData {
```

Comment:
the name “get” sounds like it should be a getter to me (though I understand why it must be an action – it commits mutations)… can we find an alternative verb? maybe “assemble”?

Reply:
Done in that other PR that will be uploaded soon™